recipe/meta.yaml

```diff
@@ -11,18 +11,16 @@ source:
 
 build:
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
+  noarch: python
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.4
```

Comment:
Do we need to go so low? I would have said a minimum of 3.7, there are a lot of things we would to take advantage of since 3.4. In my personal recipes I put in >= 3.9.

Reply:
That's what I found in [setup.py](https://github.com/brightway-lca/brightway2-calc/blob/2.0.DEV13/setup.py#L36).